src/main/java/com/gmail/nossr50/commands/party/teleport/PtpAcceptCommand.java

```diff
@@ -49,6 +49,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
         }
 
         if (mcMMO.p.getGeneralConfig().getPTPCommandWorldPermissions()) {
+            // TODO FOLIA - target is not on the same thread as sender/player - unsafe
```

Comment:
I would like these "unsafe" operations to be addressed, or at least an explanation for why they cannot be addressed.

Reply:
This was already briefly discussed in Discord DMs. I don't really remember everything I looked at as it's been a while ^^. Most likely due to this method, or an upstream method requiring/expecting immediate returns of data that would necessitate both ticking regions to be synchronized. This is obviously not how Folia works but would require a re-write of some major parts under mcMMO to solve.

To clarify, these issues were already present. I just commented them as potentially troublesome for future reference and maintenance. The PR is not focused on fixing absolutely everything under folia but rather improving some blatantly wrong parts. I also happened to add comments to stuff that could be an issue going forward.